contexts/v1.jsonld

```diff
@@ -2,7 +2,7 @@
   "@context": {
     "@protected": true,
 
-    "StatusList2021Credential": {
+    "StatusListCredential2021": {
```

Comment:
This shouldn't be changed -- this is a type of `Credential` not a `Credential2021`, so it should end in just `Credential`.

```suggestion
    "StatusList2021Credential": {
```

Reply:
Yes, considered that. It's strange... can't think of anywhere else that we do this. It creates this new rule that people have to remember and it's not clear when to use one vs. the other. I'd rather we stick with one rule (dates go at the end) rather than (dates sometimes go in the middle -- here's yet another complex VC thing you have to remember to determine when to do what).

Reply:
The other rule we have always had is that `Credential` is last. This would break that rule.

Reply:
Hmm, I don't think that there's a "new rule" here with date stamps. I think these have already been the rules:

1. If you want to date stamp a thing, `X`, with a year, `YEAR`, do `XYEAR`.
2. If you want a new type of `Credential` to hold / talk about `X` it should be `XCredential`. This means `XYEARCredential` for date stamped `X`'s.

I don't think we're date-stamping the credential, we're date-stamping the status list. Then, as part of how this system works, we also need a credential to put the status list in. That's what the credential is ... a credential that holds a `StatusList2021`. It's not a credential ... from 2021 (or something?) that is holding an unstamped status list. I'm not sure what the date stamp means if it's put at the end here.

I think it's better that the rules be that the date follows the thing it applies to -- which is the status list definition. The fact that there's a credential to go along with a `StatusList2021` is orthogonal, IMO.

Reply:
One thing I considered was just dropping the 2021 and just making it "StatusListCredential". Thoughts?

Reply:
> 2\. If you want a new type of `Credential` to hold / talk about `X` it should be `XCredential`. This means `XYEARCredential` for date stamped `X`'s.

In practice, this hasn't been a thing, has it? I'm thinking we could simplify things greatly by eliminating this "rule" (debatable). I don't think it's buying us much... if anything. What is it buying us?

Reply:
@msporny,

> One thing I considered was just dropping the 2021 and just making it "StatusListCredential". Thoughts?

Well, that would mean that every `StatusListCredential` would be expected to use the other `StatusList2021` terms in it. So, -1.

> In practice, this hasn't been a thing, has it? I'm thinking we could simplify things greatly by eliminating this "rule" (debatable). I don't think it's buying us much... if anything. What is it buying us?

If you're referring to `XCredential`, that's been common practice for a long while. If you're referring to `XYEARCredential` (where `X` was date stamped), then it has also been the practice, however, there is only one instance of that practice that I know of: [RevocationList2020Credential](https://w3c-ccg.github.io/vc-status-rl-2020/#revocationlist2020credential).

What it buys us is consistency in naming convention (a credential for `X` is `XCredential` -- whether or not `X` is date stamped) and not squatting on a name like `StatusListCredential` for every possible date-stamped revision of status list in the future.

Reply:
Per `CredentialStatusList2017`  and many other types seem to be ubiquitous to me. But this is a credential right? Hm.

My personal feeling is that the whole `YEAR` being part of a naming convention for a Credential is: (a) a little weird, but I get why you might want it ... (b) if present, should serve as some kind of demarcation point for a thing for the purposes of being able to have different versions of it that are "applicable from the time of `YYYY`" ... 
Like, if 'versioning' of a credential type is a worry, wouldn't you just version the schema linked with the credential type instead? 🤷🏻 

Other credentials I've observed all end in `Credential` keyword and I don't think I've seen one with a `YYYY` pattern anywhere in it. I am admittedly new to exploring every credential type though.

Seems like each `XX Data Model` specification would put in place the syntactic requirement so that a naming convention could be somewhat reliable.

Reply:
> Like, if 'versioning' of a credential type is a worry, wouldn't you just version the schema linked with the credential type instead? 🤷🏻

The challenge is that, at the end of the day, if we're dealing w/ Linked Data... that value with `YYYY` on it expands to a URL. For example, `https://w3id.org/vc/status-list#StatusList2021`... if we get rid of the `YYYY`, then it'll expand to `https://w3id.org/vc/status-list#StatusList`... forever. That means, we can never change what that means semantically, which has consequences... like, what fields are expected to go with an object of that type? That's why we year-stamp some of these things that we're fairly certain are going to change in meaning over the coming decades. A StatusList in 2021 uses a bitstring... but a StatusList in 2030 might use a quantum superposition value. So, StatusList2021 means one thing, and StatusList2030 can mean something else entirely.

Some things, like `VerifiableCredential` are so foundational (and generalized) that we have a high degree of certainty that they're not going to change. However, Ed25519Signature2018 -- we're certain is going to change as 1) we get more and more implementation experience, and 2) as the cryptographic primitives are defeated, over time.

Reply:
Interesting. I get you. Seems like `StatusList2021Credential` agrees the "most" with naming conventions I suppose. 
Other options like putting a `/vYYYY/` in the path would cause issues with resolution down the line. 
Seems weird, but I agree that this particular credential type is far from settled.

Reply:
> Seems like `StatusList2021Credential` agrees the "most" with naming conventions I suppose.

I count that as 2 for, 1 undecided... would be good to get some more opinions on this... /cc @OR13 @mprorock @mkhraisha @tplooker @mavarley @TallTed @peacekeeper

Reply:
`StatusList2021Credential` being the Credential that holds the 2021 StatusList which is made up of some number of `StatusList2021Entry` Entries *seems* to make the most sense. I think it will need some clear description of how it was arrived at, something like my first sentence, to carry forward clearly into the 2022 or 2025 or 2057 editions...

Reply:
Just so there is more consensus, `XYearCredential` seems fine to me. So `StatusList2021Credential` is ok. The naming convention reminds me of ECMA script standards such as `ES2016` which are working fine in the javascript community.